google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1053,6 +1101,42 @@ def get_iam_policy(self, client=None):
             self.add_warning(err_msg)
             return None
 
+    @cached('org_policy')
+    def get_org_policy(self, client=None):
```

Comment:
This method doesn't belong to the BQ class

`class BigqueryDataSet:...`

Reply:
Moved it under the resource implementation of Resource Manager Organization Policy.

Reply:
You can't get an org policy from an org policy, this method if needed should fall under, organization, folder and project.

Reply:
Done